public/app/features/alerting/unified/components/rule-editor/alert-rule-form/simplifiedRouting/__snapshots__/SimplifiedRuleEditor.test.tsx.snap

```diff
@@ -5,8 +5,48 @@ exports[`Can create a new grafana managed alert using simplified routing can cre
   {
     "body": {
       "interval": "1m",
-      "name": "empty-group",
+      "name": "grafana-group-1",
       "rules": [
+        {
```

Comment:
This is just added because I updated the eval group to be consistent across some tests, and that example test already had a "rule" in its mock data